src/gleam/date.gleam

```diff
@@ -0,0 +1,136 @@
+/// Date is type that is similar in functionality to the Erlang calendar module,
+/// but the types do not directly interface with the calendar module.
+/// To convert an erlang date to a Date type, use `from_tuple/1`
+///
+/// Currently this module defaults to UTC.
+///
+/// Years cannot be abreviated.  Years start from 0 and go up.  So if the year = 93,
+/// it does not refer to 1993.  This is in line with the behavior of Erlang.
+///
+import gleam/result
+import gleam/int
+
+pub type Date {
+  Date(year: Int, month: Int, day: Int)
```

Comment:
Perhaps NaiveDate would be a better name as this has no timezone information.

Reply:
I was considering a `DateTime` type that elixir had, made of date, time, and other info

Reply:
The Elixir Date type also contains calendar information. I think we would need that for this not to be considered more suitable for the Naive name.